3p/integration.js

```diff
@@ -422,6 +423,16 @@ function triggerRenderStart(opt_data) {
  */
 function observeIntersection(observerCallback) {
   // Send request to received records.
+  if (window.IntersectionObserver) {
+    const io = new window.IntersectionObserver(changes => {
+      observerCallback(changes);
+    }, {
+      threshold: DEFAULT_THRESHOLD,
+    });
+    io.observe(window.document.documentElement);
```

Comment:
`window.document.documentElement` is the best I can do here. If it does not fill up our 3p iframe, we get different intersection changeEntry between native IO and polyfill IO.